.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.24-bookworm
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    git make bash curl ca-certificates pkg-config build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+# Dev user
+RUN useradd -m -s /bin/bash goland
+
+# Use per-user GOPATH (overrides the base image's GOPATH=/go)
+ENV GOPATH=/home/goland/go
+ENV PATH=$GOPATH/bin:$PATH
+
+# Prepare workspace
+RUN mkdir -p /workspaces/mdai-gateway && chown -R goland:goland /workspaces /home/goland
```

Comment:
I'm actually not too sure if this is necessary if running devcontainer outside VSCode, cause when I'm running VSCode devcontainer, this is done automagically. I feel like Goland devcontainer does the same, but I can't recall